scp/tickets.php

```diff
@@ -92,8 +92,8 @@
     }
 
     $queue_key = sprintf('::Q:%s', ObjectModel::OBJECT_TYPE_TICKET);
-    $queue_id = $queue_id ?: @$_GET['queue'] ?: $_SESSION[$queue_key]
-        ?: $cfg->getDefaultTicketQueueId();
+    $queue_id = $queue_id ?: @$_GET['queue'] ?: $thisstaff->getDefaultTicketQueueId()
```

Comment:
I think `$_SESSION[$queue_key]` should take precedence over staff's settings - otherwise we will loose stickiness of queues.